allennlp/modules/token_embedders/elmo_token_embedder_multilang.py

```diff
@@ -0,0 +1,171 @@
+        indices of shape (batch_size, timesteps) to forward, instead
+        of character indices. If you use this option and pass a word which
+        wasn't pre-cached, this will break.
+    scalar_mix_parameters : ``List[int]``, optional, (default=None).
+        If not ``None``, use these scalar mix parameters to weight the representations
+        produced by different layers. These mixing weights are not updated during
+        training.
+    aligning_files : ``Dict[str, str]``, optional, (default={}).
+        A dictionary of language identifier to a pth file with an alignment matrix.
+    """
+
+    def __init__(self,
+                 options_files: Dict[str, str],
+                 weight_files: Dict[str, str],
+                 do_layer_norm: bool = False,
+                 dropout: float = 0.5,
+                 requires_grad: bool = False,
+                 projection_dim: int = None,
+                 vocab_to_cache: List[str] = None,
+                 scalar_mix_parameters: List[float] = None,
+                 aligning_files: Dict[str, str] = {}) -> None:
+        super(ElmoTokenEmbedderMultiLang, self).__init__()
+
+        assert options_files.keys() == weight_files.keys()
+        output_dim = -1
+        for lang in weight_files.keys():
+            name = 'elmo_%s' % lang
+            elmo = Elmo(
+                    options_files[lang],
+                    weight_files[lang],
+                    1,
+                    do_layer_norm=do_layer_norm,
+                    dropout=dropout,
+                    requires_grad=requires_grad,
+                    vocab_to_cache=vocab_to_cache,
+                    scalar_mix_parameters=scalar_mix_parameters)
+            self.add_module(name, elmo)
+
+            output_dim_tmp = elmo.get_output_dim()
+            if output_dim != -1:
+                # Verify that all ELMo embedders have the same output dimension.
+                assert output_dim_tmp == output_dim
+            output_dim = output_dim_tmp
+
+        self.output_dim = output_dim
+
+        if projection_dim:
+            self._projection = torch.nn.Linear(output_dim, projection_dim)
+            self.output_dim = projection_dim
+        else:
+            self._projection = None
+
+        for lang in weight_files.keys():
+            name = 'aligning_%s' % lang
+            aligning_matrix = torch.eye(output_dim)
+            if lang in aligning_files and aligning_files[lang] != '':
+                aligninig_path = cached_path(aligning_files[lang])
+                aligning_matrix = torch.FloatTensor(torch.load(aligninig_path))
+
+            aligning = torch.nn.Linear(output_dim, output_dim, bias=False)
+            aligning.weight = torch.nn.Parameter(
+                    aligning_matrix, requires_grad=False)
+            self.add_module(name, aligning)
+
+    def get_output_dim(self):
+        return self.output_dim
+
+    def forward(
```

Comment:
The way you're formatting these arguments is inconsistent across functions, and inconsistent with the rest of the codebase.  Ok @brendan-ai2, @joelgrus - if we can find an automatic formatter that makes minimal changes to existing code, I'm probably ok with it.